server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -286,13 +324,15 @@ export class MongoCollection<T> implements core.ICollection<T>, core.IRetryable
         return properties;
     }
 
-    private errorSanitizer(error: any) {
-        for (const key in error) {
-            if (key === errorJsonKeyToSanitize) {
-                error[key] = errorSanitizationMessage;
-            } else if (typeof error[key] === "object") {
-                this.errorSanitizer(error[key]);
-            }
+    private sanitizeError(error: any) {
+        if (error) {
+            Object.keys(error).forEach((key) => {
+                if (!errorResponseKeysAllowList.includes(key)) {
```

Comment:
Can we make this a set instead? Checking if a set has a key is O(1) while checking if an array includes an item is O(n)

 https://www.tech-hour.com/javascript-performance-and-optimization

Reply:
We probably want lower case the keys too